API/cmd/migrate.go

```
@@ -0,0 +1,81 @@
+)
+
+var migrateCmd = &cobra.Command{
+	Use:   "migrate",
+	Short: "Gère les migrations de base de données",
+}
+
+var migrateUpCmd = &cobra.Command{
+	Use:   "up",
+	Short: "Exécute toutes les migrations",
+	Run: func(cmd *cobra.Command, args []string) {
+		useful.RunMigrationCommand("up")
```

golangci-lint (1.62.2)

Error return value of `useful.RunMigrationCommand` is not checked

(errcheck)